.github/actions/check-issues-are-disabled/action.yaml

```diff
@@ -0,0 +1,25 @@
+name: Check Issues Are Disabled
+description: Checks whether the current repository has disabled the issues feature.
+
+outputs:
+  issues-disabled:
+    description: Whether the issues are disabled or not.
```

Comment:
Please clarify the exact values that will be returned, since everything is weakly typed.